docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/npu-device/remote-tensor-api-npu-plugin.rst

```diff
@@ -0,0 +1,137 @@
+by exporting an NT handle on Windows, or DMA-BUF System Heap on Linux and passing that pointer as the
+``shared_buffer`` member to the ``remote_tensor(..., shared_buffer)`` create function. They allow you
+to avoid any memory copy overhead when plugging OpenVINO™ inference into an existing NPU pipeline.
+
+Supported scenario by the Remote Tensor API:
+
+* The NPU plugin context and memory objects can be constructed from low-level device, display, or memory handles and used to create the OpenVINO™ ``ov::CompiledModel`` or ``ov::Tensor`` objects.
+
+Class and function declarations for the API are defined in the following file: ``openvino/runtime/intel_npu/level_zero/level_zero.hpp``
+
+The most common way to enable the interaction of your application with the Remote Tensor API is to use user-side utility classes
+and functions that consume or produce native handles directly.
+
+Context Sharing Between Application and NPU Plugin
+###########################################################
+
+NPU plugin classes that implement the ``ov::RemoteContext`` interface are responsible for context sharing.
+Obtaining a context object is the first step in sharing pipeline objects.
+The context object of the NPU plugin directly wraps Level Zero context, setting a scope for sharing the
+``ov::RemoteTensor`` objects. The ``ov::RemoteContext`` object is retrieved from the NPU plugin.
+
+Once you have obtained the context, you can use it to create ``ov::RemoteTensor`` objects.
+
+Getting RemoteContext from the Plugin
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+To request the current default context of the plugin, use one of the following methods:
+
+.. tab-set::
+
+   .. tab-item:: Get context from Core
+      :sync: get-context-core
+
+      .. doxygensnippet:: docs/articles_en/assets/snippets/npu_remote_objects_creation.cpp
+          :language: cpp
+          :fragment: [default_context_from_core]
+
+   .. tab-item:: Get context from compiled model
+      :sync: get-context-compiled-model
+
+      .. doxygensnippet:: docs/articles_en/assets/snippets/npu_remote_objects_creation.cpp
+          :language: cpp
+          :fragment: [default_context_from_model]
+
+Memory Sharing Between Application and NPU Plugin
+###########################################################
+
+The classes that implement the ``ov::RemoteTensor`` interface are the wrappers for native API
+memory handles (which can be obtained from them at any time).
+
+To create a shared tensor from a native memory handle, use dedicated ``create_tensor``, ``create_l0_host_tensor``, or ``create_host_tensor``
+methods of the ``ov::RemoteContext`` sub-classes.
+``ov::intel_npu::level_zero::LevelZero`` has multiple overloads methods which allow to wrap pre-allocated native handles with the ``ov::RemoteTensor``
+object or request plugin to allocate specific device memory.
+For more details, see the code snippets below:
+
+
+.. tab-set::
+
+   .. tab-item:: Wrap native d13
```

Comment:
`Wrap native d13` - this d13 seems unrelated. Can you remove it or change it to `handle`?

Reply:
good catch, thank you!